### ISSUE TITLE
v8.1.3.4 tag is missing from build_tools repo

### ISSUE THREAD

**Issue by adriangibanelbtactic** [ONLYOFFICE/build_tools#877]
Your debian/Ubuntu repo has the `8.1.3-4` version of `onlyoffice-documentserver` package.

There is not an v8.1.3.4 to be found on the build_tools repo as it used to happen before.

**Steps to reproduce the problem**:
```
sudo apt update
sudo apt-cache show onlyoffice-documentserver | less
```

```
Package: onlyoffice-documentserver
Priority: optional
Section: web
Installed-Size: 1622495
Maintainer: Ascensio System SIA <support@onlyoffice.com>
Architecture: amd64
Version: 8.1.3-4
Depends: debconf (>= 0.5) | debconf-2.0, adduser, ca-certificates, coreutils, curl, libasound2, libcairo2, libcurl3 | libcurl4, libcurl3-gnutls, libgtk-3-0, libstdc++6 (>= 4.8.4), libxml2, libxss1, libxtst6, logrotate, nginx-extras (>= 1.3.13), postgresql-client (>= 9.1) | mysql-client | mysql-community-client | mariadb-client, pwgen, ttf-mscorefonts-installer, xvfb, openssl, zlib1g
Filename: pool/main/o/onlyoffice-documentserver/onlyoffice-documentserver_8.1.3_amd64.deb
Size: 610652568
MD5sum: 79bbac8ee06b1ff48d4af92ec1550f52
SHA1: 8e2e11c24d3be4e2783155dac68edf97526393f4
SHA256: 8c2c1c5dbf7b67dfd9a1f28b627789c4a02769913f1f0be6df818b42efc1a143
SHA512: 7e3e5678dcd02e60cd54472daec8f18d3f66c1232e7ba5761f5d3766e32271ca6e3705b0683ebd2805f394edfeeaf246b995fc29621afa9cecff60f9665b022e
Description: Online editors for text documents, spreadsheets, and presentations
 Document Server is an open-source office suite that includes
 collaborative online editors for text documents, spreadsheets,
 and presentations.
Description-md5: 99450664b334c80b23604012ed3f65fa
```

Then you visit: https://github.com/ONLYOFFICE/build_tools/releases/tag/v8.1.3.4 and it's a 404 - non found page because the appropiated tag has not been pushed there.

**build_tools base repo version**: N/A

**Operating system**: N/A

**Compiler version**: N/A

**Comment by igwyd** [comment]
Hello @adriangibanelbtactic, sorry for late reply. The tag was removed due to the build being moved by DevOps developers. This will not happen again in the next release.

**Comment by adriangibanelbtactic** [comment]
Thank you @igwyd . It's nice to know that this was not expected in the first place.

Is it ok to assume that the `8.1.3-4` deb version is mapped to the `v8.1.3.4` tag in the github repo?

Or is there a better way of knowning (maybe checking the package contents?) which tags is mapped to?

Thank you.

**Comment by igwyd** [comment]
As far as I know, the tags here are matched with the tags of the released packages.

**Comment by adriangibanelbtactic** [comment]
> This will not happen again in the next release.

For this issue to be properly closed we would need v8.1.3.4 tags put into place in all of the repos so that the 8.1.3-4 binary package has its source code available (Required by GPLv2 and many other licenses).

In any case as this won't, hopefully, be happening in the next release I'll close this issue.
